src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -49,7 +49,12 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     handleUserDisabledWidget() {
         this.toggleWidget(true);
         this.hasUserDisabledWidget = true;
-        // TODO: dispatch an event to form renderer to clear the token from the data import record
+        fireEvent(
```

Comment:
Would it be worth pulling this into its own function and call that dispatcher function here and in line 67 below?

Reply:
Updated! @psadie-sfdo